### PR TITLE
Bump Compass Runtime Agent and end to end tests

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- Bump Compass Runtime Agent and end to end tests

### CODE REVIEW

**Review by Maladie — DISMISSED** [review]

File: tests/end-to-end/external-solution-integration/chart/external-solution/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ containerRegistry:
 
 image:
   dir: ""
-  version: PR-9354
```
Comment (k15r) [inline tests/end-to-end/external-solution-integration/chart/external-solution/values.yaml:6]: PR-9354 was merged in commit 415d68866
would it not make sense to use the image created by this commit?

File: resources/compass-runtime-agent/values.yaml [hunk 2]
```diff
@@ -3,9 +3,9 @@ global:
     containerRegistry:
       path: eu.gcr.io/kyma-project
     runtimeAgent:
-      version: "PR-9593"
+      version: "470796a1"
     runtimeAgentTests:
-      version: "PR-9569"
```
Comment (k15r) [inline resources/compass-runtime-agent/values.yaml:8]: PR-9569 was merged in commit cda9f3805

File: resources/compass-runtime-agent/values.yaml [hunk 3]
```diff
@@ -3,9 +3,9 @@ global:
     containerRegistry:
       path: eu.gcr.io/kyma-project
     runtimeAgent:
-      version: "PR-9593"
```
Comment (k15r) [inline resources/compass-runtime-agent/values.yaml:6]: PR-9593 was merged in commit c03413362

**Review by k15r — CHANGES_REQUESTED** [review]

File: tests/end-to-end/external-solution-integration/chart/external-solution/values.yaml (hunk 1 above)
Comment (franpog859) [reply tests/end-to-end/external-solution-integration/chart/external-solution/values.yaml:6]: Done! It should make more sense right now ^^ Do you know maybe if we can check this information only from the Container Registry side?

**Review by k15r — APPROVED** [review]

**Review by Maladie — APPROVED** [review]

**Pull request merged**